main/main.cpp

```diff
@@ -3170,6 +3171,14 @@ bool Main::iteration() {
 	frames++;
 	Engine::get_singleton()->_process_frames++;
 
+	recent_frametimes[frames % 100] = ticks_elapsed;
+	// Determine the frame that took the most time to render in the last 100 rendered frames.
+	int frametime_1_percent_low = 1;
+	for (int i = 0; i < 100; i++) {
+		frametime_1_percent_low = MAX(frametime_1_percent_low, recent_frametimes[i]);
+	}
+	Engine::get_singleton()->_fps_1_percent_low = 1'000'000.0 / frametime_1_percent_low;
```

Comment:
> There's an interesting comment on line 3173 about FPS reporting. Based on that, shouldn't this code be inside the IF from line 3173 (the if below)?

1% low FPS reporting is designed to be updated as often as possible, rather than just once per second. This makes the value less stable when you print out its value in `_process`, but it makes changes more reactive, which I think is better overall.

The editor performance monitor and `--print-fps` will still only update the displayed value once per second due to how they work (see the graph screenshot in OP).

In the long term, we could look into adding a smoothing algorithm like https://github.com/godotengine/godot/pull/63356, but I think this is better left for a future PR.

> Suggestion: Since it is already printing the FPS below (L3178 and 3181), WDYT aboud add the 1% low FPT there too?

That's a good idea. I implemented this :slightly_smiling_face: